src/site/layouts/campaign_landing_page.drupal.liquid

```diff
@@ -317,12 +317,51 @@
     </div>
 
     <!-- Connect with us -->
+    {% assign socialLinksObject = fieldClpConnectWithUs.entity.fieldSocialMediaLinks.platformValues | jsonToObj %}
```

Comment:
I wonder why this wasn’t JSON to begin with

Reply:
ya it's like that all over the place, it's a good question for the cms team